packages/non-deployable/token-allocation/src/config/index.ts

```diff
@@ -38,10 +38,8 @@ export const adaptersV2: v2AdapterImplementation[] = [
 ]
```

Comment:
It doesn't let me put the comment higher but there's a ref to `coinmetrics` in the v2 list still